cli/azd/resources/scaffold/templates/main.bicept

```diff
@@ -79,4 +110,7 @@ output AZURE_RESOURCE_EVENT_HUBS_ID string = resources.outputs.AZURE_RESOURCE_EV
 {{- if .ServiceBus}}
 output AZURE_RESOURCE_SERVICE_BUS_ID string = resources.outputs.AZURE_RESOURCE_SERVICE_BUS_ID
 {{- end}}
+{{- if .AiFoundryProject }}
```

Comment:
Let's also output `AZURE_RESOURCE_AI_PROJECT_ID` as well -- this is how we understand Azure resources for `show` and other future commands.